setup.py

```diff
@@ -33,7 +33,7 @@
                   'quicklook': ['matplotlib', 'cartopy', 'pillow'],
                   'dask': ['dask>=0.16.1']}
 
-test_requires = []
+test_requires = ['pytroll-schedule']
```

Comment:
Should this maybe be added as an extra requirement too?

Reply:
I'm looking at integrating the boundary stuff really soon, so I don't know